src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,56 @@
+package core.basesyntax;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.handler.BalanceOperationHandler;
+import core.basesyntax.handler.PurchaseOperationHandler;
+import core.basesyntax.handler.ReturnOperationHandler;
+import core.basesyntax.handler.ShopOperationHandler;
+import core.basesyntax.handler.SupplyOperationHandler;
+import core.basesyntax.service.CsvReaderService;
+import core.basesyntax.service.CsvWriterService;
+import core.basesyntax.service.DataProcessorService;
+import core.basesyntax.service.DataReaderService;
+import core.basesyntax.service.DataWriterService;
+import core.basesyntax.service.FruitShopDataProcessorService;
+import core.basesyntax.strategy.ShopOperationStrategy;
+import core.basesyntax.strategy.ShopOperationStrategyImpl;
+import core.basesyntax.utility.FruitQuantityChecker;
+import core.basesyntax.utility.Operation;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Main {
+    private static final String INPUT_CSV = "src/main/resources/input.csv";
+    private static final String REPORT_CSV = "src/main/resources/report.csv";
+    private static final int INITIAL_VALUE = 0;
+
+    public static void main(String[] args) {
+        Map<Operation, ShopOperationHandler> shopOperationHandlerMap = new HashMap<>();
+        shopOperationHandlerMap.put(Operation.BALANCE,
+                new BalanceOperationHandler());
+        shopOperationHandlerMap.put(Operation.RETURN,
+                new ReturnOperationHandler());
+        shopOperationHandlerMap.put(Operation.SUPPLY,
+                new SupplyOperationHandler());
+        shopOperationHandlerMap.put(Operation.PURCHASE,
+                new PurchaseOperationHandler());
+
+        Storage.FRUIT_STORAGE.put("banana", INITIAL_VALUE);
+        Storage.FRUIT_STORAGE.put("apple", INITIAL_VALUE);
+
```

Comment:
Sorry, I wasn't clear enough previously.
This lines should be deleted
```suggestion
```

Reply:
Done